python/paddle/device/__init__.py

```diff
@@ -132,7 +141,7 @@ def is_compiled_with_xpu():
     return core.is_compiled_with_xpu()
 
 
-def XPUPlace(dev_id):
+def XPUPlace(dev_id: int):
```

Comment:
```suggestion
if TYPE_CHECKING:
    from paddle import XPUPlace
def XPUPlace(dev_id: int) -> XPUPlace:
```

上面有个 `IPUPlace` 也标一下吧 ～

Reply:
收到